samples/gitchurn/src/gitChurnMain/kotlin/GitRepository.kt

```diff
@@ -60,4 +60,20 @@ class GitRepository(val location: String) {
             }
         }
     }
+
+    fun myCommits() = memScoped {
```

Comment:
How is it different from `commits`?

Reply:
My version uses new GitRevwalk class instance to work with libgit2 revwalk objects. In old `commits` it was called directly. I see, that it should be renamed, but I don't know how to name it properly.

Reply:
So both `commits` and `myCommits` are supposed to produce the same result?
Why have both then?